src/server.ts

```diff
@@ -98,5 +90,7 @@ run.on(gptscript.RunEventType.CallProgress, data => {
   process.stdout.write(data.output[0].content.slice(prev.length))
   prev = data.output[0].content
 })
+rmSync(context.sessionDir, { recursive: true, force: true })
+rmSync(noJSContext.sessionDir, { recursive: true, force: true })
```

Comment:
I figured the best time to delete the directories would be during GPTScript execution, while we are waiting for it.

Reply:
I should also add, we delete them so that they don't just accumulate over time. Each one is usually at least 20MB I think, so I don't want to leave them sitting around.

Reply:
Is it worth putting these comments into the code?

Reply:
Yeah good call. I'll go do that

Reply:
Done